_includes/data/bcda_v2_data.html

```diff
@@ -0,0 +1,170 @@
+<p>
+    Version 2 of the Beneficiary Claims Data API (BCDA V2) is now available as of Summer 2021. 
+    To support industry alignment, BCDA V2 conforms to the <a href="https://hl7.org/fhir/R4/" class="in-text__link">FHIR R4 specification</a> and follows the <a href="http://hl7.org/fhir/us/carin-bb/history.html" class="in-text__link">CARIN Implementation Guide</a>. 
+    Below is a high level summary of data differences to expect between BCDA V1 (which supports FHIR version STU3) and BCDA V2 (which supports FHIR R4), 
+    focusing on the three primary resources provided in the API:  ExplanationOfBenefit, Patient and Coverage. 
+    In general, the differences for each resources can be grouped into two categories: 
+    1.) differences due to changes in the FHIR specification between STU3 and R4 and 
+    2.) differences due to following a different implementation guide. 
+    For more information on specific API-related changes in BCDA V2, please visit the <a href="/guide.html" class="in-text__link">Getting Started</a> Page. 
+</p>
+<h2>
+    Specification Based Changes
+</h2>
+<h3>
+    Explanation Of Benefit
+</h3>
+<p>
+    The following table summarizes changes to the EOB resource due to changes between versions 3 (STU3) and 4 (R4) of the FHIR specification.
+</p>
+<p>
+    For details, see the <a href="http://www.hl7.org/fhir/explanationofbenefit.html#resource" class="in-text__link">FHIR Explanation of Benefit resource</a> page, and select the "R3 Diff" tab under section 13.10.3 "Resource Content"
+</p>
+<table style="width:100%">
+    <tr>
+        <th>Change Type</th>
+        <th>Description</th>
+        <th>Examples</th>
+    </tr>
+    <tr>
+        <td valign=top>Consolidated/Removed Elements</td>
+        <td valign=top>Some elements were consolidated and/or removed. For instance, in STU3 there were two elements (Eob.Organization and Eob.Provider) that could represent the party responsible for the claim.  In R4, this has been consolidated into one element (Eob.provider)</td>
+        <td id="cex" valign=top>
```

Comment:
```suggestion
        <td class="cex" valign=top>
```
Ideally an ID only appears once on a page whereas a class can be applied to multiple elements. Most browsers will still apply the styles, but this could present issues. Recommend changing this to a class.